### PR TITLE
[Rector] Apply PHPUnitSetList::REMOVE_MOCKS

### PR DESCRIPTION
**Pull request by samsonasik**
Apply PHPUnitSetList::REMOVE_MOCKS to remove unnecessary mock calls.

**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

File: tests/system/Debug/Toolbar/Collectors/DatabaseTest.php [hunk 1]
```diff
@@ -23,9 +23,7 @@ final class DatabaseTest extends CIUnitTestCase
     public function testDisplay(): void
     {
         /** @var MockObject&Query $query */
-        $query = $this->getMockBuilder(Query::class)
-            ->disableOriginalConstructor()
-            ->getMock();
+        $query = $this->createMock(Query::class);
```
Comment (samsonasik) [inline tests/system/Debug/Toolbar/Collectors/DatabaseTest.php:26]: it seems cause error:

```bash


1) CodeIgniter\Debug\Toolbar\Collectors\DatabaseTest::testDisplay
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-'ROOTPATH/tests/system/Debug/Toolbar/Collectors/DatabaseTest.php:35'
+'ROOTPATH/tests/system/Debug/Toolbar/Collectors/DatabaseTest.php:33'
```

Comment (samsonasik) [reply tests/system/Debug/Toolbar/Collectors/DatabaseTest.php:26]: Fixed with update line as it read `__FILE__` https://github.com/codeigniter4/CodeIgniter4/pull/6033/commits/b2f228f00292226bcca32153fc4a96d32526dcf9

**Comment by samsonasik** [comment]
All green 🎉

**Comment by kenjis** [comment]
Is there documentation for the rule?

**Comment by samsonasik** [comment]
current documentation is list of rules, not by set yet https://github.com/rectorphp/rector-phpunit/blob/main/docs/rector_rules_overview.md

The list of remove mock rules can be checked at https://github.com/rectorphp/rector-phpunit/blob/main/config/sets/remove-mocks.php

**Comment by kenjis** [comment]
Thanks! I got the document structure.

**Review by kenjis — APPROVED** [review]

**Pull request merged**